app/(gcforms)/[locale]/(form administration)/form-builder/[id]/edit/components/elements/element-dialog/ElementDialog.tsx

```diff
@@ -8,18 +8,28 @@ import { Groups } from "@lib/hooks/form-builder/useElementOptions";
 import { ElementFilters } from "./ElementFilters";
 import { Dialog, useDialogRef } from "@formBuilder/components/shared/Dialog";
 import { ListBox } from "@formBuilder/components/shared/ListBox";
+import { useFeatureFlags } from "@lib/hooks/useFeatureFlags";
 
 export type SelectedGroupState = {
   group: Groups | "all";
   ref: React.RefObject<HTMLElement>;
 };
 
-const filterElementsByGroup = (elements: ElementOption[], selectedGroup: Groups | "all") => {
+const filterElementsByGroup = (
+  elements: ElementOption[],
+  selectedGroup: Groups | "all",
+  featureFlags: { [key: string]: boolean }
+) => {
+  const excludeAddressComplete = featureFlags["addressComplete"] ? "address" : "addressComplete";
+
```

Comment:
not sure about filtering by featureFlags inside a function intended to filterElementsByGroup.

this filter should probably be applied inside useElementOptions instead where featureFlags are used to enable/disable repeatingSets and fileInput.